state_change_block/spec.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Description": "Maintains a **state** and when **state** changes, a signal is notified that contains the `state` and `prev_state`.\nPersistence is used to maintain the **state**. If `use_persistence` is enabled, that value will populate the initial state and _not_ the `initial_state` configuration value. Any new states (from new groups) will use the configured `initial_state` for their initial state though",
+    "Properties": {
+        "backup_interval (seconds=600)": " Interval at which state is saved to disk.",
+        "use_persistence": " (bool) Whether to load the `initial_state` from persistence",
+        "exclude": " Select whether you want to exclude other signals. If checked, the only output will be `state` and `prev_state`. If not checked, `state` and `prev_state` will be appended onto the incoming signal.",
+        "group_by": " (expression) What to group the signals by. A different state will be maintained for each group",
+        "state_expr": " (expression) Property that evaluates to state. If the expression cannot be evaluated, the state will not change.",
+        "initial_state": " (expression) The state when none has been set by an incoming signal. If the `initial_state` is a python expression, it is evaluated at configuration. For example, if the `initial_state` is configured as `{{ datetime.datetime.utctime() }}`, the value of `initial_state` will the be time at configuration."
+    },
+    "Commands": {
+        "current_state (parameter": " group): Gets the current state for a given group. Response is a dictionary with `state` and `group` keys. If `group` is not specified, then a list is returned with a dictionary for each group. If the specified group does not exist, then an empty dictionary is returned."
+    },
+    "Version": "1.0.0",
```

Comment:
Specs have 3-digit versions? What does a patch update to a spec mean? I was thinking that if a block release had version `2.3.4` it meant it met spec `2.3` and has had 4 patch version updates.